src/Elastic.Apm.SmartSql/Elastic.Apm.SmartSql.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>Elastic.Apm.SmartSql</RootNamespace>
+    <AssemblyName>Elastic.Apm.SmartSql</AssemblyName>
+    <AssemblyVersion>1.0.0</AssemblyVersion>
+    <InformationalVersion>1.0.0-beta1</InformationalVersion>
+    <FileVersion>1.0.0</FileVersion>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\build\elasticapm.snk</AssemblyOriginatorKeyFile>
+    <DelaySign>false</DelaySign>
+    <PackageVersion>1.0.0-beta1</PackageVersion>
+    <Authors>Elastic and contributors</Authors>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <PackageId>Elastic.Apm.SmartSql</PackageId>
+    <Copyright>2019 Elasticsearch BV</Copyright>
+    <PackageLicenseUrl>https://github.com/elastic/apm-agent-dotnet/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/elastic/apm-agent-dotnet</PackageProjectUrl>
+    <PackageIconUrl>https://raw.githubusercontent.com/elastic/apm-agent-dotnet/master/build/nuget-icon.png</PackageIconUrl>
+    <Description>Elastic APM for SmartSql. This package contains auto instrumentation for SmartSql. See: https://github.com/elastic/apm-agent-dotnet/tree/master/docs</Description>
+    <PackageTags>apm, monitoring, elastic, elasticapm, analytics, entiryframeworkcore, efcore, SmartSql</PackageTags>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="SmartSql" Version="4.0.3" PrivateAssets="All" />
```

Comment:
It's worth targeting the lowest possible version. If there is an older one that has all the types we need in this project, we should use that one - I'm not familiar with SmartSql, so maybe this is the oldest we can use here. This would mean Elastic APM would support it from `4.0.3`. Feel free to decide if this is something worth following up on.

Reply:
Yes.SmartSql version `4.0.3` is the lowest possible version for the types needed in the project. 
Smart Sql is an ORM similar to mybatis in Java.